doc/examples/viz_surfaces.py

```diff
@@ -0,0 +1,99 @@
+"""
+========================================
+Visualize surfaces; load/save, get/set and update vtkPolyData.
+========================================
+
+import usefull functions and dipy utils 
+"""
+from __future__ import division, print_function, absolute_import
+import numpy as np
+
+# and dipy tools
+import dipy.io.vtk as io_vtk
+import dipy.viz.utils as ut_vtk
+from dipy.viz import window
+
+# Conditional import machinery for vtk
+# Allow import, but disable doctests if we don't have vtk
+from dipy.utils.optpkg import optional_package
+vtk, have_vtk, setup_module = optional_package('vtk')
+
+
+"""
+generate a empty vtkPolyData
+"""
+my_polydata = vtk.vtkPolyData()
+
+
+"""
+generate a cube with vertices and triangles numpy array
+"""
+my_vetices = np.array([[0.0,  0.0,  0.0],
+                       [0.0,  0.0,  1.0],
+                       [0.0,  1.0,  0.0],
+                       [0.0,  1.0,  1.0],
+                       [1.0,  0.0,  0.0],
+                       [1.0,  0.0,  1.0],
+                       [1.0,  1.0,  0.0],
+                       [1.0,  1.0,  1.0]])
+
+my_triangles = np.array([[0,  6,  4],
+                         [0,  2,  6],
+                         [0,  3,  2],
+                         [0,  1,  3],
+                         [2,  7,  6],
+                         [2,  3,  7],
+                         [4,  6,  7],
+                         [4,  7,  5],
+                         [0,  4,  5],
+                         [0,  5,  1],
+                         [1,  5,  7],
+                         [1,  7,  3]])
+
+
+"""
+set vertices and triangles in poly data
+"""
+ut_vtk.set_polydata_vertices(my_polydata, my_vetices)
+ut_vtk.set_polydata_triangles(my_polydata, my_triangles)
+
+
+"""
+save polydata
+"""
+file_name = "my_cube.vtk"
+io_vtk.save_polydata(my_polydata, file_name)
+print("save surface :", file_name)
+
+"""
+load polydata
+"""
+cube_polydata = io_vtk.load_polydata(file_name)
+
+
+"""
+add color based on vertices position
+"""
+cube_vertices = ut_vtk.get_polydata_vertices(cube_polydata)
```

Comment:
Here again will not render correctly. Please check if the tutorial renders correctly. If you do not know how let me know.